llvm/test/Transforms/SimplifyCFG/X86/sink-common-code.ll

```diff
@@ -1975,13 +1984,13 @@ define void @store_with_non_sunk_gep(i1 %cond, ptr %p.a, ptr %p.b, i64 %a, i64 %
 ; CHECK:       if:
 ; CHECK-NEXT:    call void @dummy()
 ; CHECK-NEXT:    [[GEP_A:%.*]] = getelementptr i8, ptr [[P_A:%.*]], i64 [[A:%.*]]
```

Comment:
For reference, where is the code that decides we can't sink this GEP?

Reply:
This is because sinking the GEP would require two phis, while only one is allowed, see https://github.com/llvm/llvm-project/blob/f322f4a55e8a60b996a0a9f0b3fe924c7af2cb1b/llvm/lib/Transforms/Utils/SimplifyCFG.cpp#L2363